core/src/main/java/com/linecorp/armeria/common/ContentTooLargeExceptionBuilder.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+/**
+ * Builds a new {@link ContentTooLargeException}.
+ */
+public final class ContentTooLargeExceptionBuilder {
+    private long transferred;
+    private long delta;
+    private long limit;
```

Comment:
Let's default to `-1`, so we can represent the case where any of these properties are unknown.